scrapers/Traxxx.py

```diff
@@ -0,0 +1,100 @@
+import re, sys, copy, json
+import requests
+
+import py_common.log as log
+from traxxx_interface import TraxxxInterface
+
+def main():
+  global traxxx
+
+  mode = sys.argv[1]
+  traxxx = TraxxxInterface()
+  fragment = json.loads(sys.stdin.read())
+
+  data = None
+
+  log.info(mode)
+
+  if mode == 'scene_name':
+    data = scene_by_name(fragment)
+  if mode == 'scene_url':
+    data = scene_query_fragment(fragment)
+  if mode == 'scene_query_fragment':
+    data = scene_query_fragment(fragment)
+  if mode == 'scene_fragment':
+    data = scene_fragment(fragment)
+
+
+  if mode == 'performer_lookup':
+    data = performer_lookup(fragment)
+  if mode == 'performer_fragment':
+    data = performer_fragment(fragment)
+  if mode == 'performer_url':
+    data = performer_url(fragment)
+
+
+  if data:
+    # log.info(json.dumps(data))
+    print(json.dumps(data))
+  else:
+    print("")
+
+def search_traxxx_for_scene(fragment):
+  title = fragment.get("title")
+  if not title:
+    title = fragment.get("name")
+  if not title:
+    return
+  return traxxx.search_scenes(title)
+
+# Return a list of scenes from a search
+def scene_by_name(fragment):
+  scenes = search_traxxx_for_scene(fragment)
+  return [traxxx.parse_to_stash_scene_search(s) for s in scenes]
```

Comment:
A minor nitpick here
If no scenes are found an error is printed
We could leave it like that or do something
```python
  if not scenes:
    return [{}]
  return [traxxx.parse_to_stash_scene_search(s) for s in scenes]
```
or even
```python
  if not scenes:
    return [{"title": "No Scene found"}]
  return [traxxx.parse_to_stash_scene_search(s) for s in scenes]
```
I am not sure what is would be better ....

Reply:
Ideally it would return whatever stash is expecting as a null result which I believe is one of the following:
```python
print(json.dumps(None))
print(json.dumps([]))
print(json.dumps({}))
```